ui/src/main/webapp/src/app/components/project-list/project-list.component.ts

```diff
@@ -56,12 +54,6 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     ngOnInit():any {
         this.updateSort();
         this.getMigrationProjects();
-        this._refreshIntervalID = setInterval(() => this.getMigrationProjects(), 30000);
```

Comment:
To be honest, I disagree.
Title of this PR says "T537 Add description to MigrationProject" and this certainly does not add description and is not related to it in any way.

I would like to keep PRs with just one clear purpose and as simple as possible. I see potential problems with having PRs which do A, B, C and D. It could potentially introduce some bug and if it is touching whole bunch of unrelated things, finding commit which introduced the bug will become nightmare.